src/components/Onboarding/steps/Analytics.js

```diff
@@ -67,13 +69,15 @@ class Analytics extends PureComponent<StepProps, State> {
     const { nextStep, t, onboarding } = this.props
     const { analyticsToggle, sentryLogsToggle } = this.state
 
+    const model = getDeviceModel(onboarding.deviceModelId)
+
     return (
       <FixedTopContainer>
         <TrackPage
           category="Onboarding"
           name="Analytics"
           flowType={onboarding.flowType}
-          deviceType={onboarding.isLedgerNano ? 'Nano S' : 'Blue'}
+          deviceType={model ? model.productName : ''}
```

Comment:
we don't need to do `model ?` because `getDeviceModel` have the guarantee to return a `DeviceModel` if i'm correct